src/qt/bitcoingui.cpp

```diff
@@ -401,7 +401,7 @@ void BitcoinGUI::createActions()
     openInfoAction = new QAction(tr("&Information"), this);
     openInfoAction->setStatusTip(tr("Show diagnostic information"));
     openRPCConsoleAction = new QAction(tr("&Debug console"), this);
```

Comment:
17096: Wasn't the purpose of this PR to change Debug -> Node
(also probably missing change in debugwindow.ui)

Reply:
@PastaPastaPasta pls see https://github.com/dashpay/dash/pull/4896#discussion_r953178746 (+ we changed the title to `Tools` long time ago)